bHY6yh-d91548/cod-pol7d7_bHY6yh-d91548.xml

```diff
@@ -0,0 +1,111 @@
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#K" n="199-r"/>
+            <cb ed="#K" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d91548"><!-- p4l6-r11 -->
+            <head xml:id="bHY6yh-d91548-Hd1e108">Rubrica 11</head>
+            <head xml:id="bHY6yh-d91548-Hd1e111" type="question-title">De conseruatione aut separatione matrimonii</head>
+            <p xml:id="bHY6yh-d91548-d1e114"> 
+              De conseruatione 
+              <lb ed="#K" n="73"/>aut separatione matrimonii rubrica xi
+            </p>
+            <p xml:id="bHY6yh-d91548-d1e121">
+              <lb ed="#K"/>¶ <ref xml:id="bHY6yh-d91548-Rd1e118">Mattheus in euangelio capitulo 19</ref> <quote xml:id="bHY6yh-d91548-Qd1e127">Quod deus 
+              <lb ed="#K"/>coniunxit homo non separet</quote>
+            </p>
+            <p xml:id="bHY6yh-d91548-d1e133">
+              <lb ed="#K"/>/ <ref xml:id="bHY6yh-d91548-Rd1e133">Idem no dixit marcus capitulo 10</ref> 
+              <lb ed="#K"/>¶ <quote xml:id="bHY6yh-d91548-Qd1e137">Moyses /Ad duritiam 
+              <lb ed="#K"/>cordis nostri permisit vobis 
+              <lb ed="#K"/>dimittere vxores vestras /Qui<lb ed="#K" break="no"/>cumque 
+              dimiserit vxorem suam excepta 
+              <lb ed="#K"/>causa fornicationis et aliam 
+              <lb ed="#K"/>duxerit mechabitur et qui dimissam: 
+                  <pb ed="#K" n="199-v"/>
+                  <cb ed="#K" n="a"/>
+              <lb ed="#K" n="1"/>duxerit mechatur</quote>
+            </p>
+            <p xml:id="bHY6yh-d91548-d1e152"> 
+               <lb ed="#K"/>¶ <ref xml:id="bHY6yh-d91548-Rd1e152">Idem lucas 16 capitulo</ref>  <!--no quote-->
+            </p>
+            <p xml:id="bHY6yh-d91548-d1e158">
+              <lb ed="#K"/>¶ <ref xml:id="bHY6yh-d91548-Rd1e161">Marcus in euangelio capitulo 16</ref> <quote xml:id="bHY6yh-d91548-Qd1e162">quicumque dimiserit vxorem 
+              <lb ed="#K"/>suam et aliam duxerit adul<lb ed="#K" break="no"/>terium 
+              committit super 
+              <lb ed="#K"/>eam /Et si vxor dimiserit
+              <lb ed="#K"/>virum suum et alii nupserit 
+              <lb ed="#K"/>mechatur</quote>
+            </p>
+            <p xml:id="bHY6yh-d91548-d1e176">
+              <lb ed="#K"/>¶ <ref xml:id="bHY6yh-d91548-Rd1e181">Paulus apostolus epistola i ad corinthios capitulo 7</ref> <quote xml:id="bHY6yh-d91548-Qd1e180">hiis qui 
+              <lb ed="#K"/>matrimonio iuncti sunt precipio 
+              <lb ed="#K"/>non ego sed dominus vxorem 
+              <lb ed="#K"/>a viro non discedere quod si disces<lb ed="#K" break="no"/>serit 
+              manere innupta aut 
+              <lb ed="#K"/>viro suo reconciliari /Et vir 
```

Comment:
@katvanderahe can you take a look at this suggestion, and try to separate your "/" marks from the preceding and succeeding word


```suggestion
              <lb ed="#K"/>viro suo reconciliari / Et vir 
```